pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -261,17 +254,27 @@ dnsServer:
   # The CIDR range used to allocate
   CIDR: "240.0.0.0/4" # ENV: KUMA_DNS_SERVER_CIDR
 
-# Kuma Discovery Service configuration
-kds:
-  server:
-    # Port of GRPC server that Local Kuma CP connects to
-    grpcPort: 5685
-    # Interval for regenerating Snapshot
-    refreshInterval: 1s
-    # TlsCertFile defines a path to a file with PEM-encoded TLS cert.
-    tlsCertFile: # ENV: KUMA_KDS_SERVER_TLS_CERT_FILE
-    # TlsKeyFile defines a path to a file with PEM-encoded TLS key.
-    tlsKeyFile: # ENV: KUMA_KDS_SERVER_TLS_KEY_FILE
-  client:
-    # RootCAFile defines a path to a file with PEM-encoded Root CA. Client will verify server by using it.
-    rootCaFile: # ENV: KUMA_KDS_CLIENT_ROOT_CA_FILE
+
+# ControlPlane mode
+multicluster:
+  global:
+    # The Zone status poller timeout
+    pollTimeout: 500ms # ENV: KUMA_MULTICLUSTER_GLOBAL_POLL_TIMEOUT
+    kds:
+      # Port of GRPC server that Local Kuma CP connects to
+      grpcPort: 5685 # ENV: KUMA_MULTICLUSTER_GLOBAL_KDS_GRPC_PORT
```

Comment:
this could be just `port`

Reply:
`grpcPort` is commonly used throughout the code, e.g. 'xds', 'sds', 'mads'